web/stories/django-girls-bucaramanga.rst

```diff
@@ -247,25 +247,70 @@ contigo para coordinar los detalles.
    <div class="section-guias">
 
 .. template:: bootstrap3/django-girls-guia
-   :name: Diego Fernando Acosta Ortiz
-   :image: guia-diego.png
+   :name: Andrea Pérez Becerra
+   :image: guia-dennise.png
    :place: Bucaramanga, Colombia
-   :community: Python Bucaramanga
-   :web: http://cusol.uis.edu.co/
-   :github: https://github.com/dacostaortiz/
-   :facebook: https://www.facebook.com/dfacos
+   :email: andrea.perez.be@gmail.com
+
+.. template:: bootstrap3/django-girls-guia
+   :name: Yurley Katterine Sanchez Florez
+   :image: guia-yurley.png
+   :place: Bucaramanga, Colombia
+   :community: Python España, Aeropython
+   :email: yurs.Ksf1@Gmail.com
+   :github: https://github.com/Yursksf1
+   :facebook: https://www.facebook.com/fsk.sruy
+
+.. template:: bootstrap3/django-girls-guia
+   :name: Juan Luis Cano
+   :image: guia-juanlu.png
+   :place: Madrid, España
+   :community: Python España, Aeropython, Python Madrid
+   :email: juanlu001@gmail.com
+   :web: http://pybonacci.org/
+   :github: https://github.com/juanlu001
+
+.. template:: bootstrap3/django-girls-guia
+   :name: John Jairo Roa Acuña
+   :image: guia-jhon.png
+   :place: Bogotá, Colombia
+   :community: Python Colombia, Pyladies Medellin, Python Medellin
```

Comment:
I think all of these people should have only one community. Probably, only the most representative. Otherwise, the web page it is too load.